src/pages/Main/Main.js

```diff
@@ -1,8 +1,210 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { IoIosArrowBack } from 'react-icons/io';
+import { IoIosArrowForward } from 'react-icons/io';
+import { BANNER_INFO, MAGAZINE_DATA } from './MainData';
+import todayBg from '../assets/images/insense_6.JPG';
 import './Main.scss';
 
 const Main = () => {
-  return <div>메인페이지 입니다.</div>;
+  const [productList, setProductList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/data.json', {
+      method: 'GET',
+    })
+      .then(response => response.json())
+      .then(data => setProductList(data));
+  }, []);
+
+  return (
+    <div className="Main">
+      <div className="bannerBody">
+        <div className="bannerContainer">
+          {BANNER_INFO.map(function (banner) {
+            return (
+              <div
+                key={banner.id}
+                className="mainBanner"
+                style={{ backgroundImage: `url(${banner.bannerImg})` }}
+              >
+                <div className="alignText">
+                  <span className="bannerTitle">{banner.bannerTitle}</span>
+                  <span className="bannerText">{banner.bannerText}</span>
+                </div>
+              </div>
+            );
+          })}
+        </div>
+      </div>
+      <div className="alignBox">
+        <IoIosArrowBack className="mainArrowIcon" />
+        <IoIosArrowForward className="mainArrowIcon" />
+      </div>
+      <div className="mainContents">
+        <div className="mainNotice">
+          <p className="notice">공지사항</p>
+          <div className="noticeBox">
+            <ul className="noticeList">
+              <li>2023년 3월 다다일상 챌린지 당첨자 발표</li>
+            </ul>
+            <IoIosArrowForward style={{ marginRight: '10px' }} />
+          </div>
+        </div>
+        {/** 추천 목록 캐러셀 지역 */}
+        <section className="recommendSector">
+          <div className="recommendTitle">오래 기억될 순간들</div>
+          <div className="recommendList">
+            {productList.map(function (product) {
```

Comment:
```suggestion
            {productList.map(function ({id,pdImage,pdName}) {
```
이렇게 key만 빼서 구조분해할당도 가능합니다
참고해주세요~